src/features/animations/ui/animation-empty-state.tsx

```
@@ -0,0 +1,35 @@
+        </div>
+
+        <h4 className="text-foreground/90 relative -top-5">
+          No saved animations yet
+        </h4>
+
+        <p className={S.emptyDescription()}>
+          Start by creating a folder or saving an animation.
+        </p>
+
+        <CreateAnimationCollectionDialog>
+          <button className="inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-secondary text-secondary-foreground hover:bg-secondary/80 h-10 px-4 py-2 w-[220px]">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)